src/common/types/SatoshiBig.ts

```diff
@@ -88,6 +88,10 @@ export default class SatoshiBig extends Big {
     return BigInt(this.toFixed(0));
   }
 
+  toRBTCBigIntUnsafe(): bigint {
+    return BigInt(this.mul(100_000_000_00).toFixed(0));
+  }
```

Comment:
Isn't this converting to wei? If so consider renaming. Also formatting the multiplier to match decimal places makes it clearer.
```suggestion
  toWeiBigIntUnsafe(): bigint {
    return BigInt(this.mul(10_000_000_000).toFixed(0));
  }
```

Reply:
Good catch, it it renamed to `toWeiBigIntUnsafe` ;)